security/maltrail/src/opnsense/mvc/app/views/OPNsense/Maltrail/general.volt

```diff
@@ -0,0 +1,53 @@
+{#
+
+OPNsense® is Copyright © 2014 – 2018 by Deciso B.V.
```

Comment:
also style, but let me fix that post-merge

Reply:
Didn't find a different version than this

Reply:
https://github.com/opnsense/core/blob/master/src/opnsense/mvc/app/views/OPNsense/Core/license.volt

Reply:
done